extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/menu/ContinuousTestingProcessor.java

```diff
@@ -93,7 +96,14 @@ private void registerStartMethod(LaunchModeBuildItem launchModeBuildItem) {
                 if (testSupport.isStarted()) {
                     return false; // Already running
                 } else {
-                    testSupport.start();
+                    CompletableFuture<Void> start = CompletableFuture.runAsync(() -> testSupport.start());
+                    Uni.createFrom().completionStage(start).runSubscriptionOn(Infrastructure.getDefaultExecutor()).subscribe();
```

Comment:
This won't work - nothing will happen. Worse, the thread switch is dangerous.

Reply:
Above works (so the tests starts). But I could not get it working without using runAsync. Checking now with your suggestion above.

Reply:
Because runAsync runs (it's not lazy) but it's dangerous.